convert2rhel/pkghandler.py

```diff
@@ -72,29 +73,42 @@ def call_yum_cmd(command, args="", enablerepo=None, disablerepo=None,
     """Call yum command and optionally print its output."""
     loggerinst = logging.getLogger(__name__)
 
-    cmd = "yum %s -y" % (command)
+    cmd = "yum %s -y" % command
     # disablerepo parameter must be added before the enablerepo parameter
 
     if disablerepo is None:
         disablerepo = []
     if disablerepo:
         repos = disablerepo
-    else:
+    elif tool_opts.disablerepo:
         repos = tool_opts.disablerepo
+    else:
+        repos = ['*']
     for repo in repos:
-        cmd += " --disablerepo=%s " % repo
+        cmd += " --disablerepo=%s" % repo
 
     if enablerepo is None:
         enablerepo = []
     if enablerepo:
         repos = enablerepo
     else:
         repos = tool_opts.enablerepo
+
+    if not tool_opts.disable_submgr:
+        if not system_info.submgr_enabled_pkgs:  # double check to get all RHSM enabled repo if not get yet
+            subscription.get_enabled_repos()
```

Comment:
This seems like a useless call. `subscription.get_enabled_repos()` just returns a list of repo IDs. Is this to make sure we get `ret_code` 0?

We shouldn't call this more than once if it's the latter, one call should be enough to not slow down c2r